vnext/Microsoft.ReactNative.Cxx.UnitTests/NativeModuleTest.cpp

```diff
@@ -583,6 +599,24 @@ struct SimpleNativeModule {
     provider.Add(L"const62", "MyConstant62");
   }
 
+  REACT_GET_CONSTANTS(Constant7)
+  SimpleNativeModuleConstants1 Constant7() noexcept {
+    SimpleNativeModuleConstants1 x;
```

Comment:
>; [](http://example.com/codeflow?start=34&length=1)

It would be nice to show that the best practice is initialize the struct with {}, and also have a better variable name:

    SimpleNativeModuleConstants1 constants{};